java/src/jmri/progdebugger/ProgDebugger.java

```diff
@@ -30,6 +30,7 @@
  * TODO Fully support numberformat "113.12" in ProgDebugger (used in LOCONETLNCVMODE and LOCONETBDOPSWMODE)
  *
  * @author Bob Jacobsen Copyright (C) 2001, 2007, 2013
+ * @author Andrew Crosland Copyright (C) 2021
```

Comment:
It would be good to remove this comment now when the new `readCV` method is removed from this class.